project_types/base/project.py

```diff
@@ -284,20 +284,20 @@ def handle_new_project_on_firebase(self, project_ref: FbReference):
             resultCount=0,
             groupSize=self.project.group_size,
             projectId=self.project.old_id or str(self.project.id),
+            name=self.project.generated_name,
             projectDetails=self.project.description or "n/a",
-            name=self.project.name,
-            # FIXME(tnagorra): Fill these when implemented
-            projectNumber=1,
-            projectRegion="",
-            projectTopic="",
+            projectNumber=self.project.project_number,
+            projectRegion=self.project.region,
+            projectTopic=self.project.topic,
+            # NOTE: projectTopicKey is not obsolete
```

Comment:
```suggestion
            # NOTE: projectTopicKey is obsolete
```

My bad. projectTopicKey is obsolete